### ISSUE TITLE
batchnorm() unexpected keyword argument

### ISSUE THREAD

**Issue by leo-p** [thtrieu/darkflow#17]
First, thanks a lot for your work! I'm trying to get it to run but fail at the second step.

`./flow --h` does what it's intended to do but when I move on to 
`./flow --model cfg/yolo-tiny.cfg --load bin/yolo-tiny.weights` then I get the following error:

`batch_norm() got an unexpected keyword argument 'initializers'`

Which relates to the function `batchnorm` in `convolution.py`:
```
def batchnorm(self, layer, inp):
	if not self.var:
		temp = (inp - layer.w['moving_mean'])
		temp /= (np.sqrt(layer.w['moving_variance']) + 1e-5)
		temp *= layer.w['gamma']
		return temp
	else: return slim.batch_norm(inp, 
		center = False, scale = True, epsilon = 1e-5,
		initializers = layer.w, scope = self.scope,
		is_training = layer.h['is_training'])
```
And more precisely `initializers = layer.w`

After looking around, for instance [here](https://github.com/tensorflow/models/blob/master/inception/inception/slim/ops.py), it seems that `slim.batch_norm()` doesn't use an `initializers` argument. Is there anything I'm missing? Should I install a specific version of tensorflow to run this?

**Comment by leo-p** [comment]
Obviously I would stumble upon your answer [here](https://github.com/thtrieu/darkflow/issues/15) just after posting the issue...
I can't seem to find a way to delete it but if you can feel free!

The answer was: "for 0.12.0-rc0, `initializers` should be `param_initializers`"
